src/pymovements/dataset/dataset.py

```diff
@@ -231,6 +231,30 @@ def load_precomputed_reading_measures(self) -> None:
             self.paths,
         )
 
+    def split_gaze_data(
+            self,
+            by: list[str] | str,
```

Comment:
let's use `Sequence` here from `collections`, this way it's more in line with the polars signature: https://docs.pola.rs/api/python/stable/reference/dataframe/api/polars.DataFrame.partition_by.html

Reply:
done